modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -827,6 +867,16 @@ def validate_response_schema(response, contract_name)
 
         SchemaContract::ValidationInitiator.call(user:, response:, contract_name:)
       end
+
+      def eps_appointments_service
+        @eps_appointments_service ||=
+          Eps::AppointmentService.new(current_user)
+      end
+
+      def eps_appointments
+        @eps_appointments ||=
+          eps_appointments_service.get_appointments
+      end
     end
```

Comment:
Isn't this supposed to be in the controller for fetching EPS appointments?  Did it get included in the PR accidentally?

Reply:
We moved all the logic to the service level, and we're passing in "eps" in the includes object instead of relying on the controller level now

Reply:
Changed current_user to user